### DISCUSSION TITLE
Inserting Table Footer for Count/Totals

### DISCUSSION THREAD

**Discussion by vmosoti** [rappasoft/laravel-livewire-tables#450] [Q&A] [answered]
Hi Everyone,

Anyone got an idea on how to implement this?

For example I have column 'Amount' and would like to add the total of 'Amounts' of filtered rows at the footer. Any way to do this?

**Comment by rappasoft** [comment, marked as answer]
I do not believe we have a current way to do this without you completely copying and modifying the views.

**Reply by rappasoft** [reply]
@vmosoti I have added this in v1.16

https://rappasoft.github.io/laravel-livewire-tables-docs/columns/Footer-row

**Reply by vmosoti** [reply]
awesome ❤️

**Reply by mahdijafaridev** [reply]
@rappasoft I had installed v1.13 and wanted to upgrade to the latest version, and there was no command to do this. so I changed 1.13 to 1.20 in composer.json and then run  `composer update`. The problem is that `linkTo()`, `->footer()` ... and these new updates does not work.
I need to have a footer row that calculates the total amount. what can I do?

**Comment by ntraykov** [comment]
Hello. Is there a way to get the grand total of all rows from all pages? The only thing that I can think of was to create a custom view and put the totals there. The problem that I face now is that I don't know how to apply the filters to the query. I dag into the code and I saw that there is a method called `applyFilters`. I cannot use it because it applies the filters to the main builder which includes pagination. Could you please advice me how to do that? Thanks in advance!
